add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -1001,7 +1002,48 @@ const kMessageHandlers = {
    */
   TelemetryAddValue: function(message, reply) {
     let [histogramId, value] = message.data;
-    Services.telemetry.getHistogramById(histogramId).add(value);
+    if (histogramId === "LOOP_MAU") {
+      let buckets;
+      let pref = "mau.";
+      this.GetAllConstants(null, result => buckets = result.LOOP_MAU_TYPE);
```

Comment:
Please change this to `buckets = LOOP_MAU_TYPE;` - the call to `GetAllConstants` is unnecessary.